DabraHalleloEndaAbunaSamuel/DHAS024.xml

```diff
@@ -0,0 +1,428 @@
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="127">27</measure>                                  
+                               <dimensions type="outer" unit="mm">
+                                        <height>220</height>
+                                        <width>190</width>
+                                        <depth>90</depth>
+                                   
+                                    </dimensions>
+                                    
+                                </extent>   
+                                <foliation>Fragments of at least five manuscripts foliated throughout.</foliation>
+                                <condition key="deficient">Fragments of at least five destroyed manuscriptsplaced together, with a piece of cardboard as back board.</condition>
+                            </supportDesc>
+                            
+                        </objectDesc>
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1">Binding is missing.</decoNote>                                                      
+                            </binding>
+                        </bindingDesc>
+                        
+                        </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1550" notAfter="1630">Second half of the 16th to early 17th century</origDate>.        
+                            Fragments of at least five various manuscripts, put together.
+                        </origin>
+                    </history>
+                        <additional>
+                            
+                            <adminInfo>
+                                <recordHist>
+                                    <source/>                                  
+                                          
+                                </recordHist>
+                            </adminInfo>
+                        </additional>
+                        
+                         
+                         <msPart xml:id="p1">
+                             <msIdentifier>
+                                 <idno/>
+                             </msIdentifier>
+                             <msContents>
+                                 <summary/>            
+                         <msItem xml:id="p1_i1">
+                             <locus from="1ra" to="6vb"/>
+                             <locus from="9ra" to="22vb"/>
+                             <title type="incomplete" cert="low">Collection of hymns</title>
```

Comment:
Can this be made more specific? What kind of hymns?

Reply:
I will try, in another turn.